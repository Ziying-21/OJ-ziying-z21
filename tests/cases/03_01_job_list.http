###
# Send request:
GET http://127.0.0.1:12345/jobs HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 2
# content-type: application/json
# access-control-expose-headers: content-type
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# access-control-allow-credentials: true
# date: Fri, 09 Sep 2022 02:19:02 GMT
# 
# []
###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":1,"source_code":"fn main() {let mut line1 = String::new();std::io::stdin().read_line(&mut line1).unwrap();let a: i32 = line1.trim().parse().unwrap();let mut line2 = String::new();std::io::stdin().read_line(&mut line2).unwrap();let b: i32 = line2.trim().parse().unwrap();println!(\"{}\", a + b);}","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 1137
# content-type: application/json
# access-control-expose-headers: content-type
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# access-control-allow-credentials: true
# date: Fri, 09 Sep 2022 02:19:02 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Waiting","time":0},{"id":1,"info":"","memory":0,"result":"Waiting","time":0},{"id":2,"info":"","memory":0,"result":"Waiting","time":0},{"id":3,"info":"","memory":0,"result":"Waiting","time":0},{"id":4,"info":"","memory":0,"result":"Waiting","time":0},{"id":5,"info":"","memory":0,"result":"Waiting","time":0},{"id":6,"info":"","memory":0,"result":"Waiting","time":0},{"id":7,"info":"","memory":0,"result":"Waiting","time":0},{"id":8,"info":"","memory":0,"result":"Waiting","time":0},{"id":9,"info":"","memory":0,"result":"Waiting","time":0},{"id":10,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-09T02:19:02.953Z","id":0,"result":"Waiting","score":0.0,"state":"Queueing","submission":{"contest_id":0,"language":"Rust","problem_id":1,"source_code":"fn main() {let mut line1 = String::new();std::io::stdin().read_line(&mut line1).unwrap();let a: i32 = line1.trim().parse().unwrap();let mut line2 = String::new();std::io::stdin().read_line(&mut line2).unwrap();let b: i32 = line2.trim().parse().unwrap();println!(\"{}\", a + b);}","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/0 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 1222
# content-type: application/json
# access-control-expose-headers: content-type
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# access-control-allow-credentials: true
# date: Fri, 09 Sep 2022 02:19:03 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":171517},{"id":1,"info":"","memory":0,"result":"Accepted","time":61865},{"id":2,"info":"","memory":0,"result":"Accepted","time":1990},{"id":3,"info":"","memory":0,"result":"Accepted","time":1419},{"id":4,"info":"","memory":0,"result":"Accepted","time":1933},{"id":5,"info":"","memory":0,"result":"Accepted","time":1843},{"id":6,"info":"","memory":0,"result":"Accepted","time":1583},{"id":7,"info":"","memory":0,"result":"Accepted","time":1429},{"id":8,"info":"","memory":0,"result":"Accepted","time":1433},{"id":9,"info":"","memory":0,"result":"Accepted","time":1413},{"id":10,"info":"","memory":0,"result":"Accepted","time":1891}],"created_time":"2022-09-09T02:19:02.953Z","id":0,"result":"Accepted","score":100.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":1,"source_code":"fn main() {let mut line1 = String::new();std::io::stdin().read_line(&mut line1).unwrap();let a: i32 = line1.trim().parse().unwrap();let mut line2 = String::new();std::io::stdin().read_line(&mut line2).unwrap();let b: i32 = line2.trim().parse().unwrap();println!(\"{}\", a + b);}","user_id":0},"updated_time":"2022-09-09T02:19:03.217Z"}
###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":2,"source_code":"fn main() {let mut data: Vec<u32> = Vec::new();loop {let mut line = String::new();if std::io::stdin().read_line(&mut line).unwrap() == 0 {break;}data.push(line.trim().parse().unwrap());}data.sort_by_key(|&x| x);data.into_iter().for_each(|x| println!(\"{}\", x));}","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 1122
# content-type: application/json
# access-control-expose-headers: content-type
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# access-control-allow-credentials: true
# date: Fri, 09 Sep 2022 02:19:03 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Waiting","time":0},{"id":1,"info":"","memory":0,"result":"Waiting","time":0},{"id":2,"info":"","memory":0,"result":"Waiting","time":0},{"id":3,"info":"","memory":0,"result":"Waiting","time":0},{"id":4,"info":"","memory":0,"result":"Waiting","time":0},{"id":5,"info":"","memory":0,"result":"Waiting","time":0},{"id":6,"info":"","memory":0,"result":"Waiting","time":0},{"id":7,"info":"","memory":0,"result":"Waiting","time":0},{"id":8,"info":"","memory":0,"result":"Waiting","time":0},{"id":9,"info":"","memory":0,"result":"Waiting","time":0},{"id":10,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-09T02:19:03.966Z","id":1,"result":"Waiting","score":0.0,"state":"Queueing","submission":{"contest_id":0,"language":"Rust","problem_id":2,"source_code":"fn main() {let mut data: Vec<u32> = Vec::new();loop {let mut line = String::new();if std::io::stdin().read_line(&mut line).unwrap() == 0 {break;}data.push(line.trim().parse().unwrap());}data.sort_by_key(|&x| x);data.into_iter().for_each(|x| println!(\"{}\", x));}","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/1 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 1208
# content-type: application/json
# access-control-expose-headers: content-type
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# access-control-allow-credentials: true
# date: Fri, 09 Sep 2022 02:19:04 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":233006},{"id":1,"info":"","memory":0,"result":"Accepted","time":283786},{"id":2,"info":"","memory":0,"result":"Accepted","time":2082},{"id":3,"info":"","memory":0,"result":"Accepted","time":2051},{"id":4,"info":"","memory":0,"result":"Accepted","time":1901},{"id":5,"info":"","memory":0,"result":"Accepted","time":2020},{"id":6,"info":"","memory":0,"result":"Accepted","time":2157},{"id":7,"info":"","memory":0,"result":"Accepted","time":1903},{"id":8,"info":"","memory":0,"result":"Accepted","time":2157},{"id":9,"info":"","memory":0,"result":"Accepted","time":2351},{"id":10,"info":"","memory":0,"result":"Accepted","time":2007}],"created_time":"2022-09-09T02:19:03.966Z","id":1,"result":"Accepted","score":100.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":2,"source_code":"fn main() {let mut data: Vec<u32> = Vec::new();loop {let mut line = String::new();if std::io::stdin().read_line(&mut line).unwrap() == 0 {break;}data.push(line.trim().parse().unwrap());}data.sort_by_key(|&x| x);data.into_iter().for_each(|x| println!(\"{}\", x));}","user_id":0},"updated_time":"2022-09-09T02:19:04.550Z"}
###
# Send request:
POST http://127.0.0.1:12345/jobs HTTP/1.1
content-type: application/json

{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); }","user_id":0}
###
# Got response:
# HTTP 200
# content-length: 377
# content-type: application/json
# access-control-expose-headers: content-type
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# access-control-allow-credentials: true
# date: Fri, 09 Sep 2022 02:19:04 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Waiting","time":0},{"id":1,"info":"","memory":0,"result":"Waiting","time":0}],"created_time":"2022-09-09T02:19:04.981Z","id":2,"result":"Waiting","score":0.0,"state":"Queueing","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); }","user_id":0},"updated_time":""}
###
# Send request:
GET http://127.0.0.1:12345/jobs/2 HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 426
# content-type: application/json
# access-control-expose-headers: content-type
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# access-control-allow-credentials: true
# date: Fri, 09 Sep 2022 02:19:05 GMT
# 
# {"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":120437},{"id":1,"info":"","memory":0,"result":"Accepted","time":65367}],"created_time":"2022-09-09T02:19:04.981Z","id":2,"result":"Accepted","score":100.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); }","user_id":0},"updated_time":"2022-09-09T02:19:05.172Z"}
###
# Send request:
GET http://127.0.0.1:12345/jobs HTTP/1.1


###
# Got response:
# HTTP 200
# content-length: 2860
# content-type: application/json
# access-control-expose-headers: content-type
# vary: Origin, Access-Control-Request-Method, Access-Control-Request-Headers
# access-control-allow-credentials: true
# date: Fri, 09 Sep 2022 02:19:05 GMT
# 
# [{"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":171517},{"id":1,"info":"","memory":0,"result":"Accepted","time":61865},{"id":2,"info":"","memory":0,"result":"Accepted","time":1990},{"id":3,"info":"","memory":0,"result":"Accepted","time":1419},{"id":4,"info":"","memory":0,"result":"Accepted","time":1933},{"id":5,"info":"","memory":0,"result":"Accepted","time":1843},{"id":6,"info":"","memory":0,"result":"Accepted","time":1583},{"id":7,"info":"","memory":0,"result":"Accepted","time":1429},{"id":8,"info":"","memory":0,"result":"Accepted","time":1433},{"id":9,"info":"","memory":0,"result":"Accepted","time":1413},{"id":10,"info":"","memory":0,"result":"Accepted","time":1891}],"created_time":"2022-09-09T02:19:02.953Z","id":0,"result":"Accepted","score":100.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":1,"source_code":"fn main() {let mut line1 = String::new();std::io::stdin().read_line(&mut line1).unwrap();let a: i32 = line1.trim().parse().unwrap();let mut line2 = String::new();std::io::stdin().read_line(&mut line2).unwrap();let b: i32 = line2.trim().parse().unwrap();println!(\"{}\", a + b);}","user_id":0},"updated_time":"2022-09-09T02:19:03.217Z"},{"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":233006},{"id":1,"info":"","memory":0,"result":"Accepted","time":283786},{"id":2,"info":"","memory":0,"result":"Accepted","time":2082},{"id":3,"info":"","memory":0,"result":"Accepted","time":2051},{"id":4,"info":"","memory":0,"result":"Accepted","time":1901},{"id":5,"info":"","memory":0,"result":"Accepted","time":2020},{"id":6,"info":"","memory":0,"result":"Accepted","time":2157},{"id":7,"info":"","memory":0,"result":"Accepted","time":1903},{"id":8,"info":"","memory":0,"result":"Accepted","time":2157},{"id":9,"info":"","memory":0,"result":"Accepted","time":2351},{"id":10,"info":"","memory":0,"result":"Accepted","time":2007}],"created_time":"2022-09-09T02:19:03.966Z","id":1,"result":"Accepted","score":100.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":2,"source_code":"fn main() {let mut data: Vec<u32> = Vec::new();loop {let mut line = String::new();if std::io::stdin().read_line(&mut line).unwrap() == 0 {break;}data.push(line.trim().parse().unwrap());}data.sort_by_key(|&x| x);data.into_iter().for_each(|x| println!(\"{}\", x));}","user_id":0},"updated_time":"2022-09-09T02:19:04.550Z"},{"cases":[{"id":0,"info":"","memory":0,"result":"Compilation Success","time":120437},{"id":1,"info":"","memory":0,"result":"Accepted","time":65367}],"created_time":"2022-09-09T02:19:04.981Z","id":2,"result":"Accepted","score":100.0,"state":"Finished","submission":{"contest_id":0,"language":"Rust","problem_id":0,"source_code":"fn main() { println!(\"Hello World!\"); }","user_id":0},"updated_time":"2022-09-09T02:19:05.172Z"}]
