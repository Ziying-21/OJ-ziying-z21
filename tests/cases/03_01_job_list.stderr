[2022-09-09T02:19:01Z INFO  actix_server::builder] Starting 8 workers
[2022-09-09T02:19:01Z INFO  actix_server::server] Actix runtime found; starting in Actix runtime
[2022-09-09T02:19:02Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET / HTTP/1.1" 404 0 "-" "-" 0.000204
[2022-09-09T02:19:02Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs HTTP/1.1" 200 2 "-" "-" 0.000126
[2022-09-09T02:19:02Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 1137 "-" "-" 0.000547
[2022-09-09T02:19:03Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/0 HTTP/1.1" 200 1222 "-" "-" 0.000365
[2022-09-09T02:19:03Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 1122 "-" "-" 0.001036
[2022-09-09T02:19:04Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/1 HTTP/1.1" 200 1208 "-" "-" 0.000413
[2022-09-09T02:19:04Z INFO  actix_web::middleware::logger] 127.0.0.1 "POST /jobs HTTP/1.1" 200 377 "-" "-" 0.001134
[2022-09-09T02:19:05Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs/2 HTTP/1.1" 200 426 "-" "-" 0.000274
[2022-09-09T02:19:05Z INFO  actix_web::middleware::logger] 127.0.0.1 "GET /jobs HTTP/1.1" 200 2860 "-" "-" 0.000227
